content/en/docs/Guides/backup/_index.md

```diff
@@ -66,14 +66,56 @@ To configure automatic backups to an [S3 bucket](https://aws.amazon.com/s3/), cr
 	"sub": {
 		"access_key_id": "$ACCESS_KEY_ID",
 		"secret_access_key": "$SECRET_ACCESS_KEY_ID",
-		"endpoint": "$ENDPOINT",
 		"region": "$BUCKET_REGION",
 		"bucket": "$BUCKET_NAME",
 		"path": "backups/db.sqlite3.gz"
 	}
 }
 ```
-`interval` is configurable and must be set to a [Go duration string](https://pkg.go.dev/maze.io/x/duration#ParseDuration), `vacuum` is optional and, if set to `true`, instructs rqlite to first [`VACUUM`](https://www.sqlite.org/lang_vacuum.html) the backup copy before it uploads it. In the example above, rqlite will check every 5 minutes if an upload is required, and do so if needed. You must also supply your Access Key, Secret Key, S3 bucket name, and the bucket's region, but setting the Endpoint is optional. The backup will be stored in the bucket at `path`, which should also be set to your preferred value. Leave all other fields as is.
+`interval` is configurable and must be set to a [Go duration string](https://pkg.go.dev/maze.io/x/duration#ParseDuration), `vacuum` is optional and, if set to `true`, instructs rqlite to first [`VACUUM`](https://www.sqlite.org/lang_vacuum.html) the backup copy before it uploads it. In the example above, rqlite will check every 5 minutes if an upload is required, and do so if needed. You must also supply your Access Key, Secret Key, S3 bucket name, and the bucket's region. The backup will be stored in the bucket at `path`, which should also be set to your preferred value. Leave all other fields as is.
+
+### Non-Amazon S3 Storage
+Provided they implement an S3-compliant API, rqlite can back up to non-Amazon cloud storage such as [Wasabi](https://wasabi.com/), [Backblaze B2](https://www.backblaze.com/cloud-storage), or self-hosted solutions such as [MinIO](https://min.io/).  This is done by specifying the `endpoint` field in the `sub` object, and, where needed, the `force_path_style` field.
+
+Wasabi supports virtual-host-style URL formats (as with native S3), but does require an explicit endpoint based on the bucket's region. The example below targets a bucket called `rqlite-kq7z9xg` in Wasabi's eu-central-1 region:
+
+```json
+{
+	"version": 1,
+	"type": "s3",
+	"interval": "5m",
+	"vacuum": false,
+	"sub": {
+		"access_key_id": "$ACCESS_KEY_ID",
+		"secret_access_key": "$SECRET_ACCESS_KEY_ID",
+		"endpoint": "s3.eu-central-1.wasabisys.com",
+		"region": "eu-central-1",
+		"bucket": "rqlite-kq7z9xg",
+		"path": "backups/db.sqlite3.gz"
+	}
+}
+```
+
+For MinIO deployments that use path-style requests (which is MinIO's default
```

Comment:
"path-style" is a bit obscure. Is there any way we can explain why it actually means? An example or link to something which helps explain?

Reply:
I could link "path-style requests" to https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html#path-style-access ?

Reply:
SGTM.

Reply:
Ack, will push an update in the morning.